### PR TITLE
chore(mojaloop#2882): update bulk transfer timeout seq diagrams

### PR DESCRIPTION
**Pull request by kleyow**
chore(mojaloop#2882):  update bulk transfer timeout seq diagrams - [https://github.com/mojaloop/project/issues/2882](https://github.com/mojaloop/project/issues/2882)

- Updated payload to 3303 error used in implementation in diagram
- Update overview seq diagram with callback examples from the bulk-api-adapter

### CODE REVIEW

File: docs/technical/central-bulk-transfers/assets/diagrams/sequence/seq-bulk-3.1.0-timeout-overview.plantuml [hunk 1]
```diff
@@ -103,13 +103,119 @@ group Transfer Expiry
         |||
         TOPIC_NOTIFICATIONS <- NOTIFY_HANDLER: Consume message
         ref over DFSP1, TOPIC_NOTIFICATIONS : Send notification to Participant (Payer)\n
+        alt Timeout before any processing
+            note left of NOTIFY_HANDLER #yellow
+                PUT /bulkTransfers/<uuid>
```
Comment (elnyry-sam-k) [inline docs/technical/central-bulk-transfers/assets/diagrams/sequence/seq-bulk-3.1.0-timeout-overview.plantuml:108]:
```suggestion
                PUT /bulkTransfers/<ID>
```

File: docs/technical/central-bulk-transfers/assets/diagrams/sequence/seq-bulk-3.1.0-timeout-overview.plantuml [hunk 2]
```diff
@@ -103,13 +103,119 @@ group Transfer Expiry
...
+                                }
+                                "transferId": <uuid>
+                            },
+                        ]
+                    }
+                }
+            end note
+        else Timeout in middle of processing
+            note left of NOTIFY_HANDLER #yellow
+                PUT /bulkTransfers/<uuid>
```
Comment (elnyry-sam-k) [inline docs/technical/central-bulk-transfers/assets/diagrams/sequence/seq-bulk-3.1.0-timeout-overview.plantuml:136]:
```suggestion
                PUT /bulkTransfers/<ID>
```

Minor edit but we can have ID everywhere I suppose..

File: docs/technical/central-bulk-transfers/assets/diagrams/sequence/seq-bulk-3.1.0-timeout-overview.plantuml [hunk 3]
```diff
@@ -103,13 +103,119 @@ group Transfer Expiry
         |||
         TOPIC_NOTIFICATIONS <- NOTIFY_HANDLER: Consume message
         ref over DFSP1, TOPIC_NOTIFICATIONS : Send notification to Participant (Payer)\n
+        alt Timeout before any processing
+            note left of NOTIFY_HANDLER #yellow
+                PUT /bulkTransfers/<uuid>
+                {
+                    headers: <bulkTransferHeaders>,
+                    body: {
+                        "bulkTransferState": "COMPLETED",
```
Comment (elnyry-sam-k) [inline docs/technical/central-bulk-transfers/assets/diagrams/sequence/seq-bulk-3.1.0-timeout-overview.plantuml:112]:
```suggestion
                        "bulkTransferState": "COMPLETED",
                        "completedTimestamp": "2022-08-18T01:00:24.407Z",
```

File: docs/technical/central-bulk-transfers/assets/diagrams/sequence/seq-bulk-3.1.0-timeout-overview.plantuml [hunk 4]
```diff
@@ -103,13 +103,119 @@ group Transfer Expiry
...
+                    }
+                }
+            end note
+        else Timeout in middle of processing
+            note left of NOTIFY_HANDLER #yellow
+                PUT /bulkTransfers/<uuid>
+                {
+                    headers: <bulkTransferHeaders>,
+                    body: {
+                        "bulkTransferState": "COMPLETED",
```
Comment (elnyry-sam-k) [inline docs/technical/central-bulk-transfers/assets/diagrams/sequence/seq-bulk-3.1.0-timeout-overview.plantuml:140]:
```suggestion
                        "bulkTransferState": "COMPLETED",
                        "completedTimestamp": "2022-08-18T01:00:24.407Z",
```

**Comment by elnyry-sam-k** [comment]
Thanks Kevin, overall looks good..

Made couple of minor suggestions (change to use ID in the URI and adding completedTimestamp)

**Comment by kleyow** [comment]
@elnyry-sam-k done.

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**